docs/adr/0005-use-google-python-style-guide.md

```diff
@@ -0,0 +1,38 @@
+# Use Google Python style guide with minimal refinements
+
+Technical Story: https://github.com/theupdateframework/tuf/issues/1128
+
+## Context and Problem Statement
+
+The Secure Systems Lab code style guide, which has been used for most of the
+code base, has become outdated. Through the upcoming rewrite, we have the
+chance to ignore consistency considerations with existing code style and can
+choose a more standard and up-to-date style guide.
+
+## Decision Drivers
+
+* Flaws in original Secure Systems Lab style guide
+* Curating a complete custom style guide is time consuming
+* Well-established style rules lower contribution barrier
```

Comment:
Nit: You might argue this is already covered by the "lower contribution barrier" driver, but this was a major factor for me:

```suggestion
* Well-established style rules lower contribution barrier
* Custom style is not supported by default in common tooling (i.e. editors
  and linters)
```